src/slices/releaseNoteSlice.js

```diff
@@ -29,45 +29,46 @@ export const releaseNoteReducer = createReducer(
   }
 );
 
-// THUNKS 
+// THUNKS
 export function fetchReleaseNoteById(id) {
-    return dispatch => {
-      dispatch(getByIdPending({id}));
-      return Axios.get("/releasenote/" + id)
-        .then(response => {
-          dispatch(getByIdSuccess({data: response.data, id}));
-        })
-        .catch(error => {
-          getByIdError({error, id});
-        });
-    };
-  }
-  
-  export function fetchReleaseNotes() {
-    return dispatch => {
-      dispatch(getPending());
-      return Axios.get("/releasenote")
-        .then(res => {
-          dispatch(getSuccess(res.data));
-        })
-        .catch(error => {
-          getError(error);
-        });
-    };
-  }
-  
-  export function putReleaseNote(id, note) {
-    return dispatch => {
-      dispatch(savePending({id}));
-      return Axios.put("/releasenote/" + id, note)
-        .then(response => {
-          dispatch(saveSuccess({data: response.data, id}));
-        })
-        .catch(error => {
-          saveError({error, id});
-        });
-    };
-  }
+  return dispatch => {
+    dispatch(getByIdPending({ id }));
+    return Axios.get("/releasenote/" + id)
+      .then(response => {
+        dispatch(getByIdSuccess({ data: response.data, id }));
+      })
+      .catch(error => {
+        getByIdError({ error, id });
+      });
+  };
+}
+
+export function fetchReleaseNotes(_query) {
```

Comment:
Hvorfor må "_" brukes?

Reply:
Det e vel bære sånn at vi kan bruk ordet `query`, men lign jo litt på en anna naming-convention i C#.

Reply:
Trengte berre to navn for query. Men egentlig litt dumt e gjor det slik, for e fann ut at `_` e en vanlig naming convention for private variabla